uiowa/uiowa
# Banner mobile overlay

Resolves https://github.com/uiowa/uiowa/issues/4919


## To test
1.  setup:  
  -  `git checkout banner-mobile-overlay`
  - `ddev composer install`
  - `ddev blt frontend`
  - `ddev blt ds --site=sandbox.uiowa.edu`
2.  Replicate
  - https://sandbox.uiowa.ddev.site/banners-imagesvideos-w-black-and-white-gradients
  - https://sandbox.uiowa.ddev.site/banner-backgrounds-w-gradient-dark
  - https://sandbox.uiowa.ddev.site/banner-backgrounds-w-gradient-light
3.  On each of the replicated pages above edit each banner and add the mobile stacking style. 
  - You should have 
    - the 3 original pages 
    - the 3 duplicated pages w/ each banner set to mobile stacked
5. Test the six pages on mobile and desktop and look for any colors that are unreadable like black on black. 
6. Edit each of the six pages and add a black background pattern to the layout section holding all of the banners
 - Test the six pages on mobile and desktop and look for any colors that are unreadable like black on black. 
7.  Repeat step 5 for a white, gray, and gold background color. 

Banners of the same background color as the section will have their margin/padding removed: https://github.com/uiowa/uiowa/pull/3702

### Known issues
 
You might come across a few issues like:
- Blockquote border is not visible on white gradient with gold section background: fixed on https://github.com/uiowa/uids/pull/652/commits/6d8bb2ba204a99221e921d55ee14bb913650c7b1
- Headline highlight background is the same color as the background color on nested backgrounds: https://github.com/uiowa/uids/issues/680

These are background issues related to the blockquote and headline components that will not be resolved within this pull request.   As long as the text colors are adapting and providing accessible color contrast, this should move forward.
